src/modules/dashboard/components/ColonyDecisionPreview/DetailsWidget/DetailsWidget.css

```diff
@@ -0,0 +1,46 @@
+.item {
```

Comment:
all details widget files are leftovers from rebase, they were deleted and are not used anywhere.